inference-engine/src/offline_transformations/include/json_extension/json_config_extension.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <fstream>
+#include <iostream>
+#include <type_traits>
+
+#include "common/extensions/decoder_transformation_extension.hpp"
+#include "common/frontend_defs.hpp"
+#include "nlohmann/json.hpp"
+
+namespace ov {
+namespace frontend {
+
+/// \brief JsonConfigExtension reads MO config file and delegate transformation functionality to specified
+/// transformation ID specified in the config.
+class FRONTEND_API JsonConfigExtension : public DecoderTransformationExtension {
```

Comment:
looks like `FRONTEND_API ` is not needed here since `offline_transformations` is a static library

Reply:
removed